src/app/portfolio/AssetsBreakdown.tsx

```diff
@@ -131,7 +163,7 @@ export const AssetsBreakdown: React.FC<{
                     <AssetsBreakdownRow
                       key={`${i}_${asset.name}`}
                       asset={asset}
-                      totalBalance={totalBalance}
+                      totalBalance={totalBalanceIncludingStaking} // Use the updated total balance
```

Comment:
Doesnt' the current `totalBalance` include staking balances already?

It's calculated [here](https://github.com/AdamikHQ/adamik-app/blob/1f0ccaa2c1bce34d2d668db5fff5276361271c20/src/app/portfolio/page.tsx#L133-L137) as:
```
  const totalBalance =
    availableBalance +
    stakingBalances.claimableRewards +
    stakingBalances.stakedBalance +
    stakingBalances.unstakingBalance;
```
I think it's more a bug with this calculation, but the calculation should happen before this point.
I don't think `totalBalanceIncludingStaking` should be needed.

Reply:
You are perfectly right.